content/docs/gherkin/reference.md

```diff
@@ -262,17 +267,15 @@ Example: Multiple Givens
 
 ## Background
 
-Occasionally you'll find yourself repeating the same `Given` steps in all of the scenarios in a feature.
+Occasionally you'll find yourself repeating the same `Given` steps in all of the scenarios in a `Feature` or `Rule`.
 
 Since it is repeated in every scenario, this is an indication that those steps
 are not *essential* to describe the scenarios; they are *incidental details*. You can literally move such `Given` steps to the background, by grouping them under a `Background` section.
 
-A `Background` allows you to add some context to the scenarios in the feature. It can contain one or
+A `Background` allows you to add some context to the scenarios that follow it. It can contain one or
 more `Given` steps.
 
-A `Background` is run before *each* scenario, but after any [Before hooks](/docs/cucumber/api/#hooks). In your feature file, put the `Background` before the first `Scenario`.
-
-You can only have one set of `Background` steps per feature. If you need different `Background` steps for different scenarios, you'll need to split them into different feature files.
+A `Background` is run before *each* scenario, but after any [Before hooks](/docs/cucumber/api/#hooks). In your feature file, put the `Background` before the first `Scenario`/`Example`.
```

Comment:
How would this work with rule? Can you have multiple rules per feature? If so, can you now have multiple backgrounds in a feature file. Examples would be nice.

Reply:
> Can you have multiple rules per feature?

Yep

> If so, can you now have multiple backgrounds in a feature file

Yes, in effect - but still only one background as a direct child of a feature. I think this does need some rework to communicate that better.